paper.md

```diff
@@ -129,18 +131,25 @@ Hack week outcomes, in turn, depend strongly on participants and are often a fun
+Feedback from the data science ethnographer Brittany Fiore-Gartland (REF TO BLOG POST) after the first Astro Hack Week led to the realization early on that hack weeks were particularly prone to promote impostor syndrome in participants. Impostor syndrome refers to the subjective feeling that a person is less qualified than those around them, that they are a fraud about to be found out and expelled from the community [REF]. This is exacerbated by the pressure hack weeks implicitly put on their attendance to expose their ideas to public scrutiny, find collaborators in a very short amount of time, and not least produce and present a successful hack at the end (see also this insightful blog post [source](https://medium.com/astronomy-without-stars/the-horror-of-hack-days-52c6b52cfc3b).
+Anecdotal evidence suggests that the occurrence of impostor syndrome at hack weeks is largely independent of academic seniority and objective expertise. Much of this can be attributed to the design focus on diversity, and it is one of the downsides of admitting a very heterogeneous group. Because every participant in the room will be expert in one particular subject or method, this easily leads to the subjective perception that every participant has superior knowledge than oneself. Anecdotal evidence confirms that this is true even for tutorial speakers who were specifically invited to act as experts on a given topic. The presence of impostor syndrome at a hack week is endemic and must be a major concern for any organizing committee, because of the chilling effect it tends to have on participants and the community as a whole. Impostor syndrome inhibits risk taking: participants experiencing impostor syndrome will be less likely to ask a question, to put forward an idea for a hack, to be pro-active about forming new collaborations. This, in turn, seriously threatens many of the goals of a hack week, including the successful completion of projects, lateral knowledge transfer as well as community building. 
+Thankfully, impostor syndrome can be mitigated. In minority participants, ensuring adequate representation among the participants can decrease feelings of otherness and thus reduce impostor syndrome. More generally, being open about its presence and prevalence can help participants feel more at ease (e.g. one Astro Hack Week participant remarked in a survey, _"I really appreciated the direct acknowledgement of impostor syndrome on the first day.  I think it helped ease the feeling!!"_) . Additionally, role models are very effective at encouraging positive behaviour: asking participants with prior experience at hack weeks at all academic levels to ask questions, even when they might know the answer, can help foster an inclusive environment that rewards risk taking. 
```

Comment:
My read of this is we are suggesting that openly speaking to the impostor syndrome problem can mitigate it. In my experience, talking about it at the opening is just a first step, and the problem can persist even after repeatedly identifying it over several days. I suggest changing the tone here to reflect how complex the problem is, that we are just beginning to explore how to solve it, and these are our specific strategies to address it moving forward (I have some ideas for next geohack...).